### ISSUE TITLE
Provide proper documentation for using Tooltip with selector

### ISSUE THREAD

**Issue by axelfontaine** [twbs/bootstrap#41020]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

There is currently no documentation on how to use Tooltip with 'selector' using plain JS.

### Reduced test cases

This unfortunately doesn't work:

```js
document.querySelector('body').tooltip({
  allowList: myAllowList,
  selector: '[data-bs-toggle="tooltip"]'
});
```

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Firefox

### What version of Bootstrap are you using?

5.3.3

**Comment by nwalters512** [comment]
Does the "enable tooltips" example in the docs not meet your needs?

https://getbootstrap.com/docs/5.3/components/tooltips/#enable-tooltips

```ts
const tooltipTriggerList = document.querySelectorAll('[data-bs-toggle="tooltip"]')
const tooltipList = [...tooltipTriggerList].map(tooltipTriggerEl => new bootstrap.Tooltip(tooltipTriggerEl))
```

If not, it'd be helpful to hear more about exactly what you're trying to achieve.

**Comment by axelfontaine** [comment]
@nwalters512 No, as this doesn't take care of elements added dynamically to the page later on.

**Comment by axelfontaine** [comment]
Well that doesn't really document the selector setting, which from its description, should alleviate the need to pull in another dependency.

**Comment by nwalters512** [comment]
🙈 never mind; today I learned something new about the Bootstrap API! Yes, it'd be great for this to be formally documented.

**Comment by mdo** [comment]
Fixed by #42552
